src/mock/aktivitet.ts

```diff
@@ -32,20 +32,21 @@ const aktiviteter: Aktivitet[] & JSONArray = [
         jobbStatus: null,
         ansettelsesforhold: null,
         arbeidstid: null,
-        behandlingType: null,
+        behandlingType: 'HAN MÅÅÅ PÅÅÅ BEHANDLIIIING!',
         behandlingSted: null,
         effekt: null,
         behandlingOppfolging: null,
         kanal: 'OPPMOTE',
-        erReferatPublisert: false
+        erReferatPublisert: false,
+        klokkeslett: 123
```

Comment:
Ser ut som klokkeslett er noe som egentlig kommer fra fradato, vet ikke om det er tenkt å innføre et nytt felt i backend?

Reply:
Jeg har sett litt på det: Sverger opp og ned og til siden at aktivitetsplan genererte klokkeslett slik i sine JSON-objekter, men når jeg kikket kort på herokuappen nå så fant jeg det ikke 🤔

Reply:
Der var du ja :) Ser litt raskt på aktivitetsplan og ser ut som den tar `klokkeslett` som er satt når man submitter aktiviteten også lagrer den i jsonobjektet i `fraDato` og sender til backend. Så da må vi få konvertert fram og tilbake i denne appen også. Det er litt dumt å ha samme logikken i både aktivitetsplan og dialog syns jeg så kanskje vi burde vurdere å innføre et nytt felt i backend?

Reply:
Et eget klokkeslett-felt med integer som type (minutter etter midnatt) er jo det jeg har hatt oppi hodet mitt. Nå er jeg både forvirret om det faktisk eksisterer, og også hvorvidt du synes det er en god idé, eller om du mener noe annet burde gjøres

Reply:
Feltene `klokkeslett` og `varighet` finnes ikke backend i dag dessverre, det er noe som finnes kun i aktivitetsplan frontend som kalkulerer verdiene basert på `fradato` og `tildato`. Jeg syns dog det hadde vært bedre hvis man hadde fått disse feltene mer i tråd med det du har antatt i pull requesten her. For nå så fjerner jeg det som har med varighet og klokkeslett sånn at vi kan merge også kan nestemann ta stiling til om man vil endre backend.